src/camera.cpp

```diff
@@ -269,28 +258,21 @@ void Camera::addArmInertia(f32 player_yaw, f32 frametime)
 		    following a vector, with a smooth deceleration factor.
 		*/
 
-		f32 acc_X = (m_cam_vel_old.X * (1.0f + (1.0f - m_arm_dir.X))) /
-			(20.0f / std::fabs(55.0f - m_wieldmesh_offset.X));
-		f32 acc_Y = (m_cam_vel_old.Y * (1.0f + (1.0f - m_arm_dir.Y))) /
-			(15.0f / std::fabs(-35.0f - m_wieldmesh_offset.Y));
+		f32 dec_X = 0.12f * (m_cam_vel_old.X * (1.0f +
+			(1.0f - m_arm_dir.X))) / (20.0f / gap_X);
```

Comment:
> it also permit to have a less CPU cost function (division costs far more than multiplication for CPU)

While you're correct, I am fairly sure that gcc will perform such optimizations on its own if applicable.